interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
@rzelayafavila: This **for-in without filter** issue is actually caught by eslint, because our current eslint rules do not allow it. Here is what I get when running eslint on this file:
```
  202:13  error  The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype  guard-for-in
  216:13  error  The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype  guard-for-in
  264:11  error  The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype  guard-for-in

✖ 3 problems (3 errors, 0 warnings)
```